cmd/headerSync.go

```diff
@@ -53,20 +57,21 @@ func init() {
 }
 
 func backFillAllHeaders(blockchain core.BlockChain, headerRepository datastore.HeaderRepository, missingBlocksPopulated chan int, startingBlockNumber int64) {
-	populated, err := history.PopulateMissingHeaders(blockchain, headerRepository, startingBlockNumber)
+	populated, err := history.PopulateMissingHeaders(blockchain, headerRepository, startingBlockNumber, validationWindow)
```

Comment:
This is nitpicky and nbd, but I'm wondering if we should change the `validationWindow` const to `validationWindowSize` since this constant is just an integer that is used as the size of the ValidationWindow that is created, vs. the actual window itself.